src/Util/DigitizeUtil/DigitizeUtil.js

```diff
@@ -0,0 +1,81 @@
+import OlObservable from 'ol/observable';
+
+/**
+ * This class provides some static methods which might be helpful when working
+ * with redlining functions.
+ *
+ * @class DigitizeUtil
+ */
+class DigitizeUtil {
+
+  /**
+   * Moves / translates an `OlFeature` to the given `pixel` delta in
+   * in the end with given `duration` in ms, using the given style,
+   * calling a `doneFn`.
+   *
+   * @param {OlMap} map An OlMap.
+   * @param {ol.Feature} featureToMove The feature to move.
+   * @param {Number} duration The duration in ms for the moving to complete.
+   * @param {Array<Number>} pixel Delta of pixels to move the feature.
+   * @param {ol.style.Style} style The style to use when moving the feature.
+   * @param {Function} doneFn The function to call when done.
+   *
+   * @return {String} A listener key from a postcompose event.
+   */
+   static moveFeature = (map, featureToMove, duration, pixel, style, doneFn) => {
+
+     let listenerKey;
+
+     const geometry = featureToMove.getGeometry();
+     const start = new Date().getTime();
+     const resolution = map.getView().getResolution();
+     const totalDisplacement = pixel * resolution;
+     const expectedFrames = duration / 1000 * 60;
+     let actualFrames = 0;
+     const deltaX = totalDisplacement / expectedFrames;
+     const deltaY = totalDisplacement / expectedFrames;
+
+     //eslint-disable-next-line
+     const animate = (event) => {
+       const vectorContext = event.vectorContext;
+       const frameState = event.frameState;
+       const elapsed = frameState.time - start;
+
+       geometry.translate(deltaX, deltaY);
+
+       if (vectorContext.setFillStrokeStyle && vectorContext.setImageStyle &&
+          vectorContext.drawPointGeometry) {
+         vectorContext.setFillStrokeStyle(
+           style.getFill(), style.getStroke());
+         vectorContext.setImageStyle(style.getImage());
+         if (geometry instanceof ol.geom.Point) {
```

Comment:
Here and below:

Should we import the `ol/geom/Point` and use thes in the instanceof checks?

Reply:
Yeah, indeed! And could you please remove the `ol` entry from the eslint globals (see `.eslintrc`)?

Reply:
`.eslintrc` adapted in 4a17d2a, imports fixed in 4a1ea1e